src/components/Home.js

```diff
@@ -1,21 +1,26 @@
+import { useToast } from '@1hive/1hive-ui'
 import React, { useCallback, useState } from 'react'
 import styled from 'styled-components'
+
+import { useNodeHeight } from '@hooks/useNodeHeight'
```

Comment:
What was the rationale for moving this up? 

We usually have local hooks after components.